book/src/setup.md

```diff
@@ -0,0 +1,53 @@
+# Setup & installation
+
+This section provides an opinionated guide to setting up your development environment for working with Rust and Python.
+If you're an experienced developer in either language, feel free to skip this section.
+
+## Python
+
+For macOS/Linux users, it's recommended to manage Python versions using [pyenv](https://github.com/pyenv/pyenv). `pyenv` lets you easily switch between multiple versions of Python. It's simple, unobtrusive, and follows the UNIX tradition of single-purpose tools that do one thing well.
+
+Follow the instructions from the [installation steps](https://github.com/pyenv/pyenv?tab=readme-ov-file#installation) section of the README to install `pyenv` on your system.
+
+Windows users can use [pyenv-win](https://github.com/pyenv-win/pyenv-win), a fork of `pyenv` that allows you to install and manage Windows-native Python versions.
+
+### Python version
+
+This book uses Python 3.11.x. You can install the latest minor version using `pyenv`:
```

Comment:
We could mention here that Python 3.8+ will work with these examples so if the can't, or don't want to, install new version the programs will still run.